docs/components.md

```diff
@@ -0,0 +1,37 @@
+# Samson Components
+
+## Know the basics
```

Comment:
can remove this since there is nothing non-basic here :)

Reply:
thanks.. updated.